### PR TITLE
[BE][WIKI] Delete GET resources/topic/{topicId}

### PR DESCRIPTION
**Pull request by roche-tan**

### CODE REVIEW

File: services/wiki/src/__tests__/resources/list.test.ts [hunk 1]
```diff
@@ -121,13 +125,18 @@ afterAll(async () => {
   await prisma.resource.deleteMany({
     where: { user: { id: userWithNoName?.id } },
   })
+  // await prisma.topicsOnResources.deleteMany({
```
Comment (xpan1c) [inline services/wiki/src/__tests__/resources/list.test.ts:128]: delete these comments

File: services/wiki/tsconfig.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "esModuleInterop": true,
     "skipLibCheck": true
   },
-  "include": ["src/**/*.ts"],
```
Comment (xpan1c) [inline services/wiki/tsconfig.json:16]: remove `src/__tests__/resources/byTopic.test.tst`

File: services/wiki/src/controllers/resources/list.ts [hunk 3]
```diff
@@ -74,3 +75,40 @@ export const listResources: Middleware = async (ctx: Koa.Context) => {
   ctx.status = 200
   ctx.body = parsedResources
 }
+
```
Comment (xpan1c) [inline services/wiki/src/controllers/resources/list.ts:78]: `topicId ` is handled in the query string

**Review by xpan1c — CHANGES_REQUESTED** [review]

**Review by kevinmamaqi — APPROVED** [review]

**Review by xpan1c — APPROVED** [review]
this file should be deleted: `services\wiki\src\openapi\routes\resources\id.ts`

**Pull request merged**